crash when accessing some addresses

If you try to ping some very high address like `2ff::` yggdrasil crashes. I can't provide much more info, you can try for yourself, it is very easy to reproduce the bug.

Appears to be this:
```
panic: runtime error: index out of range [32] with length 32

goroutine 9 [running]:
github.com/yggdrasil-network/yggdrasil-go/src/address.(*Address).GetKey(0xc000280610)
        /Volumes/Workspace/yggdrasil-go/src/address/address.go:132 +0x185
github.com/yggdrasil-network/yggdrasil-go/src/ipv6rwc.(*keyStore).sendToAddress(0xc00005c0c0, {0x2, 0xff, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...}, ...)
        /Volumes/Workspace/yggdrasil-go/src/ipv6rwc/ipv6rwc.go:97 +0x358
github.com/yggdrasil-network/yggdrasil-go/src/ipv6rwc.(*keyStore).writePC(0xc00005c0c0, {0xc000294004, 0x38, 0x10003})
        /Volumes/Workspace/yggdrasil-go/src/ipv6rwc/ipv6rwc.go:290 +0x385
github.com/yggdrasil-network/yggdrasil-go/src/ipv6rwc.(*ReadWriteCloser).Write(...)
        /Volumes/Workspace/yggdrasil-go/src/ipv6rwc/ipv6rwc.go:347
github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter).read(0xc00013a150)
        /Volumes/Workspace/yggdrasil-go/src/tuntap/iface.go:20 +0xba
created by github.com/yggdrasil-network/yggdrasil-go/src/tuntap.(*TunAdapter)._start
        /Volumes/Workspace/yggdrasil-go/src/tuntap/tun.go:144 +0x513
exit status 2
```

--- a/src/address/address.go
+++ b/src/address/address.go
@@ -129,7 +129,11 @@ func (a *Address) GetKey() ed25519.PublicKey {
 		bits <<= byte(idx % 8)
 		keyIdx := keyOffset + (idx - addrOffset)
 		bits >>= byte(keyIdx % 8)
-		key[keyIdx/8] |= bits
+		idx := keyIdx / 8
+		if idx >= len(key) {
+			break
+		}
+		key[idx] |= bits
 	}
 	for idx := range key {
 		key[idx] = ^key[idx]

